src/server/pathway-commons/search/index.js

```diff
@@ -16,15 +16,21 @@ const processPhrase = (phrase, collection) => {
     'refseq'
   ];
 
+  // Do NOT recognize these gene symbols
+  const symbolNameBlackList = [
```

Comment:
blacklist is checked in here:
https://github.com/PathwayCommons/app-ui/blob/development/src/server/pathway-commons/search/hgnc.js#L4

Reply:
Does it not work without this?

Reply:
nice catch - then for that file, line 15 should be AND rather than OR

Reply:
https://github.com/PathwayCommons/app-ui/blob/development/src/server/pathway-commons/search/hgnc.js#L15